scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -133,7 +133,10 @@
 	authorIdentifierScheme	LCNA		2											
 	authorIdentifierScheme	VIAF		3											
 	authorIdentifierScheme	GND		4											
-	language	Abkhaz		0											
+	authorIdentifierScheme	DAI		5
+	authorIdentifierScheme	ResearcherID		6
+	authorIdentifierScheme	ScopusID		7
+	language	Abkhaz		0
```

Comment:
I got curious about why the line for "Abkhaz" changed and it looks like the trailing tabs were removed:

<img width="1460" alt="Screen Shot 2019-11-12 at 6 45 13 AM" src="https://user-images.githubusercontent.com/21006/68669413-6f20dc80-0518-11ea-9917-a853adfe4cc9.png">

It looks like the new lines for authorIdentifierScheme don't have trailing tabs either. I assume this is fine.

It would be nice to add a file under "doc/release-notes" to explain to installations what they should do to add the new values. My understanding is that they should re-run the curl command for the citation.tsv file. This process of writing release notes is described at http://guides.dataverse.org/en/4.17/developers/making-releases.html#write-release-notes

Finally, I believe this pull request closes #3169 so the description should probably be change to that (instead of "related". On his way out the door, @djbrooke indicated that he might want to let @Venki18 make the change himself during Danny's visit to NTU but I think we should just leave "ResearcherID" in. More contributions can be made in the future. 😄

Reply:
> I believe this pull request closes #3169 so the description should probably be change to that (instead of "related")

I just made this change.

Reply:
> It would be nice to add a file under "doc/release-notes" to explain to installations what they should do to add the new values. My understanding is that they should re-run the curl command for the citation.tsv file.

@kcondon I just moved this pull request to QA without the release note mentioned above. If you want me to hop on @mderuijter 's branch and add it (or add it after merging) please let me know.